packages/server-admin-ui/src/views/appstore/appStore.scss

```
@@ -154,3 +154,381 @@
+  border: 1px solid #dee2e6;
+  transition: box-shadow 0.15s;
+}
+
+.plugin-card:hover {
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.08);
+}
+
+.plugin-card__title {
+  font-size: 1rem;
+  font-weight: 600;
+  // Long unbreakable npm package names (e.g. @marineyachtradar/signalk-plugin)
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)